data/relup_bases.eterm

```diff
@@ -0,0 +1,155 @@
+             <<"4.3.4">>,<<"4.3.5">>,<<"4.3.6">>,<<"4.3.7">>,<<"4.3.8">>,
+             <<"4.3.9">>,<<"4.3.0">>,<<"4.3.1">>,<<"4.3.10">>,<<"4.3.11">>,
+             <<"4.3.12">>,<<"4.3.13">>,<<"4.3.14">>,<<"4.3.15">>,<<"4.3.16">>,
+             <<"4.3.2">>,<<"4.3.3">>,<<"4.3.4">>,<<"4.3.5">>,<<"4.3.6">>,
+             <<"4.3.7">>,<<"4.3.8">>,<<"4.3.9">>],
+        otp => <<"23.2.7.2-emqx-3">>},
+  <<"4.3.18">> =>
+      #{from_versions =>
+            [<<"4.3.0">>,<<"4.3.1">>,<<"4.3.10">>,<<"4.3.2">>,<<"4.3.3">>,
+             <<"4.3.4">>,<<"4.3.5">>,<<"4.3.6">>,<<"4.3.7">>,<<"4.3.8">>,
+             <<"4.3.9">>,<<"4.3.0">>,<<"4.3.1">>,<<"4.3.10">>,<<"4.3.11">>,
+             <<"4.3.12">>,<<"4.3.13">>,<<"4.3.14">>,<<"4.3.15">>,<<"4.3.16">>,
+             <<"4.3.17">>,<<"4.3.2">>,<<"4.3.3">>,<<"4.3.4">>,<<"4.3.5">>,
+             <<"4.3.6">>,<<"4.3.7">>,<<"4.3.8">>,<<"4.3.9">>],
+        otp => <<"23.3.4.9-3">>},
+  <<"4.3.2">> =>
+      #{from_versions => [<<"4.3.0">>,<<"4.3.1">>,<<"4.3.0">>,<<"4.3.1">>],
+        otp => <<"23.2.7.2-emqx-3">>},
+  <<"4.3.3">> =>
+      #{from_versions =>
+            [<<"4.3.0">>,<<"4.3.1">>,<<"4.3.2">>,<<"4.3.0">>,<<"4.3.1">>,
+             <<"4.3.2">>],
+        otp => <<"23.2.7.2-emqx-3">>},
+  <<"4.3.4">> =>
+      #{from_versions =>
+            [<<"4.3.0">>,<<"4.3.1">>,<<"4.3.2">>,<<"4.3.3">>,<<"4.3.0">>,
+             <<"4.3.1">>,<<"4.3.2">>,<<"4.3.3">>],
+        otp => <<"23.2.7.2-emqx-3">>},
+  <<"4.3.5">> =>
+      #{from_versions =>
+            [<<"4.3.0">>,<<"4.3.1">>,<<"4.3.2">>,<<"4.3.3">>,<<"4.3.4">>,
+             <<"4.3.0">>,<<"4.3.1">>,<<"4.3.2">>,<<"4.3.3">>,<<"4.3.4">>],
+        otp => <<"23.2.7.2-emqx-3">>},
+  <<"4.3.6">> =>
+      #{from_versions =>
+            [<<"4.3.0">>,<<"4.3.1">>,<<"4.3.2">>,<<"4.3.3">>,<<"4.3.4">>,
+             <<"4.3.5">>,<<"4.3.0">>,<<"4.3.1">>,<<"4.3.2">>,<<"4.3.3">>,
+             <<"4.3.4">>,<<"4.3.5">>],
+        otp => <<"23.2.7.2-emqx-3">>},
+  <<"4.3.7">> =>
+      #{from_versions =>
+            [<<"4.3.0">>,<<"4.3.1">>,<<"4.3.2">>,<<"4.3.3">>,<<"4.3.4">>,
+             <<"4.3.5">>,<<"4.3.6">>,<<"4.3.0">>,<<"4.3.1">>,<<"4.3.2">>,
+             <<"4.3.3">>,<<"4.3.4">>,<<"4.3.5">>,<<"4.3.6">>],
+        otp => <<"23.2.7.2-emqx-3">>},
+  <<"4.3.8">> =>
+      #{from_versions =>
+            [<<"4.3.0">>,<<"4.3.1">>,<<"4.3.2">>,<<"4.3.3">>,<<"4.3.4">>,
+             <<"4.3.5">>,<<"4.3.6">>,<<"4.3.7">>,<<"4.3.0">>,<<"4.3.1">>,
+             <<"4.3.2">>,<<"4.3.3">>,<<"4.3.4">>,<<"4.3.5">>,<<"4.3.6">>,
+             <<"4.3.7">>],
+        otp => <<"23.2.7.2-emqx-3">>},
+  <<"4.3.9">> =>
+      #{from_versions =>
+            [<<"4.3.0">>,<<"4.3.1">>,<<"4.3.2">>,<<"4.3.3">>,<<"4.3.4">>,
+             <<"4.3.5">>,<<"4.3.6">>,<<"4.3.7">>,<<"4.3.8">>,<<"4.3.0">>,
+             <<"4.3.1">>,<<"4.3.2">>,<<"4.3.3">>,<<"4.3.4">>,<<"4.3.5">>,
+             <<"4.3.6">>,<<"4.3.7">>,<<"4.3.8">>],
+        otp => <<"23.2.7.2-emqx-3">>},
```

Comment:
we'll unlikely need to deal with 4.3
```suggestion
```

Reply:
I removed them here: https://github.com/emqx/emqx/pull/8776/commits/9bd51595957661dd02d6c3f311997978ad1e0919

While changing the script to handle multiple past OTP versions, as we are still building with OTP 23.  I'll squash the commits after everything passes.